spec/mongo/cursor_spec.rb

```diff
@@ -273,7 +279,7 @@
     context 'when the cursor is not fully iterated and is garbage collected' do
 
       let(:documents) do
-        (1..4).map{ |i| { field: "test#{i}" }}
+        (1..6).map{ |i| { field: "test#{i}" }}
```

Comment:
It's not clear at all to me why we made 3 documents before or why having 5 now is better. Why does this need to be changed?

Reply:
6 total documents mean the collection requires two getmores to iterate after the first getmore is done, which is needed for jruby as described in the comment in the next hunk.

Reply:
Ah, okay. I hadn't realized that this change was also related to that comment.